pyaedt/hfss.py

```diff
@@ -226,9 +226,18 @@ def field_setup_names(self):
     class BoundaryType(object):
         """Creates and manages boundaries."""
 
-        (PerfectE, PerfectH, Aperture, Radiation, Impedance, LayeredImp, LumpedRLC, FiniteCond, Hybrid, FEBI) = range(
-            0, 10
-        )
+        (
+            PerfectE,
+            PerfectH,
+            Aperture,
+            Radiation,
+            Impedance,
+            LayeredImp,
+            LumpedRLC,
+            FiniteCond,
+            Hybrid,
+            FEBI,
+        ) = range(0, 10)
```

Comment:
Would it make sense / be possible to switch such implementation (and others) to `enum` ?

Reply:
Yes, I will do it in a different PR